build/common.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1</TargetFrameworks>
```

Comment:
Was this necessary?

Reply:
Async enumerable for ListDirectory is only available on 2.1, 6.0 and 7.0 targets. It removed our async extension method with begin invoke.

Reply:
So we bumped it for the sole purpose of replacing a single extension method?

Reply:
Ahh. I see that method is doing some interesting things. So I guess it's worth it to get rid of that, eh?

Reply:
Yeah, was using old async wrappers to make it async before the days of await / async :)